WooCommerce/WooCommerceTests/System/WaitingTimeTrackerTests.swift

```diff
@@ -0,0 +1,101 @@
+import XCTest
+@testable import WooCommerce
+
+/// WaitingTimeTracker Unit Tests
+///
+class WaitingTimeTrackerTests: XCTestCase {
+    private var testAnalytics = TestAnalytics()
+
+    func testTimeElapsedEvaluationIsCorrect() {
+        var currentTimeCallCounter = 0.0
+
+        // Given
+        let waitingTracker = WaitingTimeTracker(trackScenario: .orderDetails,
+                                                analyticsService: testAnalytics
+        ) {
+            currentTimeCallCounter += 1
+            return currentTimeCallCounter * 10
+        }
```

Comment:
nit: We tend to write these in a single line, as long as it does not exceed the max linter max length (140 chars I bleieve)
```suggestion
        let waitingTracker = WaitingTimeTracker(trackScenario: .orderDetails, analyticsService: testAnalytics) {
            currentTimeCallCounter += 1
            return currentTimeCallCounter * 10
        }
```

Reply:
Oh, thanks for the suggestion! Fixed right here: https://github.com/woocommerce/woocommerce-ios/pull/7417/commits/168c506e00ca3f8d8f1f996f27d732eace697c34